docs/source/release/v6.0.0/sans.rst

```diff
@@ -9,4 +9,13 @@ SANS Changes
     putting new features at the top of the section, followed by
     improvements, followed by bug fixes.
 
-:ref:`Release 6.0.0 <v6.0.0>`
\ No newline at end of file
+Algorithms and instruments
+--------------------------
+
+Improvements
+############
+
+ - In :ref:`SANSILLAutoProcess <algm-SANSILLAutoProcess>`, the beam radius can be different for each distance.
```

Comment:
Please can you also mention that you've added a new TransmissionBeamRadius parameter + the default value for BeamRadius has changed from 0.05 to 0.1.

Also please could you make one other documentation update. I noticed there's a screenshot in the file drill-settings.png which shows the settings screen on the DrILL user interface. It would be useful to update because this screen contains the new TransmissionBeamRadius setting

Reply:
I updated the release note to mention the new parameter and the default value. I also changed the settings window screenshot.